metrics/core/lib/src/util/validation/validation_target.dart

```diff
@@ -0,0 +1,25 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// A class that represents a single entity used in the validation process.
```

Comment:
🏷️ 
What do you think? 
```suggestion
/// A class that represents a single entity under the validation. More precisely, this describes the value to be validated during the validation process.
```